### PR TITLE
Clarify the intro and first section

### PR DESCRIPTION
**Pull request by kevinlu11**
Made the intro and first section more concise while keeping all important information.

Renamed the first and second sections for greater clarity and organization.

### CODE REVIEW

File: files/en-us/learn/forms/styling_web_forms/index.md [hunk 1]
```diff
@@ -36,21 +36,17 @@ In the previous few articles we looked at all the HTML you'll need to create and
...
 </table>
 
-## Why is styling form widgets challenging?
+## Challenges in styling form widgets
 
-Form controls were added to HTML in [the HTML 2 specification](https://www.ietf.org/rfc/rfc1866.txt) in 1995; CSS wasn't released until late 1996, and wasn't supported very well by browsers for a few years after that. Browsers relied on the underlying operating system to manage and render form controls.
+### History
 
-And even with CSS available to style HTML, browser vendors have historically been reluctant to make form controls stylable because users were so accustomed to the visual appearance of their respective platforms. But this has changed. Website owners want form styles that fit in with their overall site design more than ever, and the web platform has changed to make this more possible.
+In 1995, [the HTML 2 specification](https://www.ietf.org/rfc/rfc1866.txt) introduced form controls (a.k.a form widgets, or form elements). But CSS wasn't released until late 1996, and wasn't supported by most browsers until years afterwards, so, in the interim, browsers relied on the underlying operating system to render form widgets.
```
Comment (sideshowbarker) [inline files/en-us/learn/forms/styling_web_forms/index.md:43]:
```suggestion
In 1995, [the HTML 2 specification](https://www.ietf.org/rfc/rfc1866.txt) introduced form controls (a.k.a form widgets, or form elements). But CSS wasn't released until late 1996, and wasn't supported by most browsers until years afterwards; so, in the interim, browsers relied on the underlying operating system to render form widgets. 
```

File: files/en-us/learn/forms/styling_web_forms/index.md [hunk 2]
```diff
@@ -80,21 +72,18 @@ Some elements can't be styled thoroughly using CSS. These include:
...
+## Styling simple form widgets
 
-As well as the basic CSS tools covered above, we've also been provided with several selectors — UI pseudo-classes — that enable styling based on the current state of the UI. We cover these in detail in the next article, [UI pseudo-classes](/en-US/docs/Learn/Forms/UI_pseudo-classes).
+The "easy-to-style" widgets in the previous section may be styled using techniques from the articles [Your first form](/en-US/docs/Learn/Forms/Your_first_form) and [CSS building blocks](/en-US/docs/Learn/CSS/Building_blocks). There are also special selectors — [UI pseudo-classes](/en-US/docs/Learn/Forms/UI_pseudo-classes) — that enable styling based on the current state of the UI.
 
-We'll walk through an example at the end of this article to give you some more ideas on basic form control styling and placement. Before we do that however, we'd like to say a few words about specific aspects of form styling that are worth knowing about.
+We'll walk through an example at the end of this article, but first, here are some special aspects of form styling that are worth knowing about.
```
Comment (sideshowbarker) [inline files/en-us/learn/forms/styling_web_forms/index.md:86]:
```suggestion
We'll walk through an example at the end of this article — but first, here are some special aspects of form styling that are worth knowing about.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Kevin, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Comment by klu22** [comment]
Thanks for reviewing! Looking forward to making more edits in the future 😁

**Pull request merged**